doc/source/cookbook.rst

```diff
@@ -105,11 +108,14 @@ Splitting
 
 .. ipython:: python
 
-   df = pd.DataFrame(
-        {'AAA' : [4,5,6,7], 'BBB' : [10,20,30,40],'CCC' : [100,50,-30,-50]}); df
+   df = pd.DataFrame({'AAA': [4, 5, 6, 7], 'BBB': [10, 20, 30, 40],
+                      'CCC': [100, 50, -30, -50]})
+   df
 
-   dflow = df[df.AAA <= 5]; dflow
-   dfhigh = df[df.AAA > 5]; dfhigh
+   dflow = df[df.AAA <= 5]
+   dflow
+   dfhigh = df[df.AAA > 5]
+   dfhigh
```

Comment:
If those are not being used later, I'd show the result directly, instead of assigning them to a variable first.